app-admin/gnome-abrt/gnome-abrt-1.4.1-r1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=no
```

Comment:
gnome-abrt-1.4.2 is in the tree, probably need to update this one

Reply:
It's just an example, I'm not going to merge it.